segfault when running head with no arguments

```
jsoares@typhoeus:~/dev/drand$ make install
go install -ldflags "-X github.com/drand/drand/cmd/drand-cli.version=`git describe --tags` -X github.com/drand/drand/cmd/drand-cli.buildDate=`date -u +%d/%m/%Y@%H:%M:%S` -X github.com/drand/drand/cmd/drand-cli.gitCommit=`git rev-parse HEAD`"
jsoares@typhoeus:~/dev/drand$ drand 
NAME:
   drand - distributed randomness service

USAGE:
   drand [global options] command [command options] [arguments...]

VERSION:
   v0.9.1-90-g422960b

COMMANDS:
   start  Start the drand daemon.
   stop   Stop the drand daemon.

   share             Launch a sharing protocol.
   generate-keypair  Generate the longterm keypair (drand.private, drand.public)for this node.

   get  get allows for public information retrieval from a remote drand node.

   util  Multiple commands of utility functions, such as reseting a state, checking the connection of a peer...
   show  local information retrieval about the node's cryptographic material. Show prints the information about the collective public key (drand.cokey), the group details (group.toml), the long-term private key (drand.private), the long-term public key (drand.public), or the private key share (drand.share), respectively.

   help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --verbose       If set, verbosity is at the debug level (default: false)
   --folder value  Folder to keep all drand cryptographic information, with absolute path. (default: "/home/jsoares/.drand")
   --help, -h      show help (default: false)
   --version, -v   print the version (default: false)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xaeaf32]

goroutine 1 [running]:
github.com/drand/drand/cmd/drand-cli.CLI.func2(0xc00007b500, 0x0, 0x0)
	/home/jsoares/dev/drand/cmd/drand-cli/cli.go:229 +0x22
github.com/urfave/cli/v2.(*App).handleExitCoder(0xc000001c80, 0xc00007b500, 0x0, 0x0)
	/home/jsoares/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:505 +0x50
github.com/urfave/cli/v2.(*App).RunContext(0xc000001c80, 0xda03a0, 0xc0000360b0, 0xc0000321b0, 0x1, 0x1, 0x0, 0x0)
	/home/jsoares/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:317 +0x74d
github.com/urfave/cli/v2.(*App).Run(...)
	/home/jsoares/go/pkg/mod/github.com/urfave/cli/v2@v2.2.0/app.go:215
main.main()
	/home/jsoares/dev/drand/main.go:11 +0x63
```

--- a/cmd/drand-cli/cli.go
+++ b/cmd/drand-cli/cli.go
@@ -226,7 +226,9 @@ func CLI() *cli.App {
 	}
 
 	app.ExitErrHandler = func(context *cli.Context, err error) {
-		fmt.Fprintf(os.Stderr, err.Error())
+		if err != nil {
+			fmt.Fprintf(os.Stderr, err.Error())
+		}
 	}
 	app.Version = version
 	app.Usage = "distributed randomness service"

